txauthor.go

```diff
@@ -16,25 +16,19 @@ import (
 )
 
 type TxAuthor struct {
-	sendFromAccount       uint32
-	destinations          []TransactionDestination
-	requiredConfirmations int32
-	wallet                *Wallet
+	sourceWallet        *Wallet
+	sourceAccountNumber uint32
+	destinations        []TransactionDestination
 }
 
-func (wallet *Wallet) NewUnsignedTx(sourceAccountNumber, requiredConfirmations int32) *TxAuthor {
+func (mw *MultiWallet) NewUnsignedTx(sourceWallet *Wallet, sourceAccountNumber int32) *TxAuthor {
```

Comment:
Can we keep `wallet.NewUnsignedTx` and have them both?
And also return nil if the `sourceAccountNumber` is invalid?

Reply:
Keeping `wallet.NewUnsignedTx` would require that the caller pass in the required confirmations. Not desired but possible. I'll look into it.

Reply:
On further thought, @C-ollins has an idea that would work better in making wallet instances access config preferences independently, will wait for such solution before re-trying to make `NewUnsignedTx` a wallet method.